File: CS/DxBlazorApplication1/DxBlazorApplication1/Data/Categories.cs
namespace DxBlazorApplication1.Data
{
    public class Categories
    {
        public Categories()
        {
            Products = new HashSet<Products>();
        }

        public int categoryId { get; set; }
        public string categoryName { get; set; }
        public string description { get; set; }

        public virtual ICollection<Products> Products { get; set; }
    }
}

File: VB/DxBlazorApplication1/DxBlazorApplication1/Data/Categories.vb
Namespace DxBlazorApplication1.Data

    Public Class Categories

        Public Sub New()
            Products = New HashSet(Of Products)()
        End Sub

        Public Property categoryId As Integer

        Public Property categoryName As String

        Public Property description As String

        Public Overridable Property Products As ICollection(Of Products)
    End Class
End Namespace
